fusor-ember-cli/app/templates/review/progress/overview.hbs

```diff
@@ -1,19 +1,34 @@
 <br />
 
-{{progress-bar model=manageContentTask name=nameSatellite isSatelliteProgressBar=true}}
+{{#if deployment.has_content_error}}
+  {{#if loadingRedeployment}}
+    <h1 style="float: left; margin: 0 15px 0 0">Content Error Occurred</h1>
+    <div style="float: left; margin-top: 2px" class="spinner spinner-md spinner-inline"></div>
+  {{else}}
+    <h1>Content Error Occurred</h1>
+    <button {{action 'abandon'}}>Abandon Deployment</button>
+    <button {{action 'redeploy'}}>Redeploy</button>
```

Comment:
@eriknelson, for RHCI it is. I'm not so sure it must be, but it works in this case where "Redeploy" (or maybe change to Re-Deploy) is on the right.